indico/modules/events/abstracts/settings.py

```diff
@@ -0,0 +1,27 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from indico.modules.events.settings import EventSettingsProxy
+
+
+abstracts_settings = EventSettingsProxy('abstracts', {
+    'description_settings': {
+        'active': True,
+        'required': True,
+        'max_chars': 0,
+        'max_words': None
```

Comment:
We used to have a `getLimitation(...)` method and most legacy code requires that we provide either `chars` or `words` IIRC. OTOH, I see that I didn't set that default in the migration script.